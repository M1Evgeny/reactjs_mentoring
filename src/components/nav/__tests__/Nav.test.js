import React from "react";
import { render } from "@testing-library/react";
import { Nav } from "..";

describe("Nav test", () => {
  test("Nav snapshot", () => {
    const { asFragment } = render(<Nav />);
    expect(asFragment()).toMatchSnapshot();
  });
});