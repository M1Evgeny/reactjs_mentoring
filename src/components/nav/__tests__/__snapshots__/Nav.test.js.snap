// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav test Nav snapshot 1`] = `
<DocumentFragment>
  <nav
    class="nav"
  />
</DocumentFragment>
`;
